install.html

```diff
@@ -7,9 +7,85 @@
 <script src=docs.js></script>
 
 <h1>Selenium Installation</h1>
+Selenium set up is quite different than other commercial tools. You don't install a tool using
+selenium binaries but set up your test automation project to use selenium binary in the langauge
+of your choice. The selenium binary you would need depends on browsers you want to run
+your tests upon.
 
 
+<h2>Using browser specific jar</h2>
+If you want to run tests in only a spcific brower then you can add browser specific dependency in
+your pom.xml file. You should add following dependency in your pom.xml file to run
+tests only on Firefox browser -
+
+
+<textarea rows="6" cols="100" style="border:none; margin-top:10px; margin-bottom:10px;">
+<dependency>
+    <groupId>org.seleniumhq.selenium</groupId>
+    <artifactId>selenium-firefox-driver</artifactId>
+    <version>2.53.1</version>
+</dependency>
+</textarea>
+
+that's all and you are ready to driver your tests in firefox browser.
+In the similar manner, if you want to run tests only in chrome browser then you can use
+following dependency -
+
+<textarea rows="6" cols="100" style="border:none; margin-top:10px; margin-bottom:10px;">
+<dependency>
+    <groupId>org.seleniumhq.selenium</groupId>
+    <artifactId>selenium-chrome-driver</artifactId>
+    <version>2.53.1</version>
+</dependency>
+</textarea>
+
 <h2>Using standalone jar</h2>
+More often than not you would need to run test on more than one browser and you would also like
+to take advantage of <a href=quick.html#selenium_grid>Selenium Grid</a> to distribute tests in
+a combination of operating systems and browser. Hence you would use selenium-java
+maven dependency in your project -
```

Comment:
agree, it belong to stand-alone jar referred in [current doc](https://seleniumhq.github.io/docs/install.html#selenium_installation). Updates on way.